### PR TITLE
Minio

### PR DESCRIPTION
**Pull request by brutskov**

### CODE REVIEW

File: client/app/containers/test-run-info/test-run-info.controller.js [hunk 1]
```diff
@@ -756,12 +756,17 @@ const testRunInfoController = function testRunInfoController(
...
             const logIsBefore = l.timestamp <= log.timestamp;
             const nextLogIsAfter = !$scope.logs[index + 1] || ($scope.logs[index + 1].timestamp > log.timestamp);
             return logIsBefore && nextLogIsAfter;
         });
 
+        if (!logToAttache && !$scope.logs.length) {
+            $scope.logs.push(log);
+            logToAttache = log;
+        }
+
```
Comment (danymill) [inline client/app/containers/test-run-info/test-run-info.controller.js:769]: revert these changes, the are outdated

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/containers/test-run-info/test-run-info.controller.js (hunk 1 above)
Comment (brutskov) [reply client/app/containers/test-run-info/test-run-info.controller.js]: done

**Review by danymill — APPROVED** [review]
